### ISSUE TITLE
After update, the user area badges area badge names are all messed up

### ISSUE THREAD

**Issue by scottcher** [NoahY/q2a-badges#8]
I can't seem to figure out how to put images to these issues forms so I'll copy-paste the text to see if that illustrates the problem.  I had just updated the Badges Plugin to resolve an issue with the Comment Voting plugin and its new badges.

This is what it's doing - looks like its messed up the end tags or something - note the addition of "> after the text.

Bronze

questions (Bronze)">Asker x 1
answers (Bronze)">Answerer x 1
comments (Bronze)">Commenter x 1
upvote (Bronze)">Nice Question x 2
upvote (Bronze)">Nice Answer x 2
times (Bronze)">Voter x 1
total upvotes (Bronze)">Liked x 1
answer selected as best answer (Bronze)">Gifted x 1
answer as best answer (Bronze)">Grateful x 1
edit (Bronze)">Editor x 1
questions (Bronze)">Reader x 1
points (Bronze)">100 Club x 1
Photogenic x 1
Autobiographer x 1
badges (Bronze)">Medalist x 1
Silver

times (Silver)">Avid Voter x 1
answers selected as best answer (Silver)">Wise x 1
edits (Silver)">Copy Editor x 1
questions (Silver)">Avid Reader x 1
points (Silver)">1,000 Club x 1
Gold

edits (Gold)">Senior Editor x 1

**Comment by scottcher** [comment]
Reverting to the previous version resolves the issue (version 3.2)

**Comment by scottcher** [comment]
Note, reverting the 3.4 update causes a problem with the names and descriptions for custom badge added as part of another plugin.  I've seen this for my own custom badge plugin as well as your comment voting plugin which includes badges.

**Comment by NoahY** [comment]
should be fixed in latest code.

**Comment by scottcher** [comment]
Latest code meaning the code available for download now or code that will be available for download soon?  The code that's available now (NoahY-q2a-badges-58712ef) is what introduced the issue for me.

**Comment by NoahY** [comment]
Sorry, for some reason it wouldn't push.  Should be there now.

**Comment by scottcher** [comment]
Just tested the new code and I'm still seeing issues.  I'm getting undefined index warnings and when I view the badges in the history plugin, they look like "upvote (Bronze)">Nice Question" - similar to how they used to look in the profile area.  I'm wondering if the display errors from history are because the event log stored outdated description formats for the badges?

I turned off the plugin before I looked at the other areas of the site where the Badges plugin creates content so I don't know if other issues were there and if the original issue was resolved.
